src/js/11-elastic-search.js

```diff
@@ -62,14 +72,56 @@ class ElasticSearch {
       .then((resultList) => {
         const currentLocation = window.location.pathname
         const locale = currentLocation.includes('en-gb') ? 'en-gb' : 'de-de'
+        const componentsTitle = translateKey(locale, 'components_title')
+        const componentFilter = this.options.filters.all[0].url_path_dir2
+        const modulesTitle = translateKey(locale, 'modules_title')
+        const moduleFilter = this.options.filters.all[1].url_path_dir4
         const resultsLabel = translateKey(locale, 'results_label')
         this.searchresults = ''
+        this.componentFacets = ''
+        this.moduleFacets = ''
+
         this.createPagination(resultList.info.meta.page.current, resultList.info.meta.page.total_pages)
         document.getElementById('searchnores').innerHTML = `${resultList.info.meta.page.total_results} ${resultsLabel}`
         resultList.results.forEach((result) => {
           this.searchresults += '<a class="the-search-result" href="' + result.data.url.raw + '"><span class="result-title">' + result.data.headings.raw[0] + '</span><span class="result-description">' + result.data.article_content.raw + '...</span><span class="result-url">' + result.data.url.raw + '</span></a>'
```

Comment:
You already have handlebars in your project. I would suggest to use it to generate this markup as well.